content/notebooks/how_to_run_mrchem.ipynb

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
<p>Add the <code>$</code> for the prompt:</p><p> </p><pre class="ql-syntax" spellcheck="false">$ export OMP_NUM_THREADS=16
$ mrchem jobname.inp
</pre><p> </p><p>Personally, I prefer to do <code>$ OMP_NUM_THREADS=N mrchem jobname.inp</code> such that the environment is not polluted.</p><p>One question you might get: how do I put <code>mrchem</code> on my <code>PATH</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MRChemSoft/multiwavelet-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I thought they would need to do this due to the way it is set up with the conda environment?